index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Damon Romano's.js Quiz</title>
+  </head>
+  <body>
+
+    How many lines tall would you like your pine tree to be?
+    <input type="text" name="Rows?" autofocus id="pineTreeHeight">
+
+    Please type the single character that will comprise your Pine Tree.
+    <input type="text" maxlength="1" name="Character?" id="pineTreeCharacter">
```

Comment:
Why are you using the name attribute? It only is required when you submit your form to a server (not using ajax). I would simply remove the attribute. Regardless, I recommend you stick with only lowercase [a-z] for the name. If you insisted, I would think about it like an object. If I were going to hold the values of the form (6 & ^) in a JSON object, i would probably choose:
```json
{
  "rows": 6,
  "character": "^"
}
```

Thus, I would want my name attribute to match the object's keys: "rows" and "character"